std/os/windows/ws2_32.zig

```diff
@@ -0,0 +1,102 @@
+usingnamespace @import("bits.zig");
+
+// TODO use GetNameInfoW?
+pub extern "ws2_32" stdcallcc fn getnameinfo(
+    pSockaddr: *const sockaddr,
+    SockaddrLength: socklen_t,
+    pNodeBuffer: [*]u8,
+    NodeBufferSize: DWORD,
+    pServiceBuffer: [*]u8,
+    ServiceBufferSize: DWORD,
+    Flags: c_int,
+) c_int;
+
+// TODO use GetAddrInfoW?
+pub extern "ws2_32" stdcallcc fn getaddrinfo(
```

Comment:
`getaddrinfo` isn't present on older versions of windows.  Not sure if we want to handle it or not, but other language like D handle this function specially because of it:
https://github.com/dlang/phobos/blob/275f3947d3542dc9a21c2730e8298d36cc424bba/std/socket.d#L298